test/backward_compatibility/check_backward_compatibility.py

```diff
@@ -79,6 +79,10 @@
     ("aten::divide", datetime.date(2021, 4, 28)),
     ("aten::_var", datetime.date(2021, 5, 28)),
     ("aten::_std", datetime.date(2021, 5, 28)),
+    ("aten::std", datetime.date(2021, 5, 28)),
```

Comment:
Last issue: these entries need to be updated since the old dates expired:

```
Jul 07 19:50:17 Broken ops: [
Jul 07 19:50:17 	aten::std_mean(Tensor self, bool unbiased=True) -> (Tensor, Tensor)
Jul 07 19:50:17 	aten::std_mean.dim(Tensor self, int[1] dim, bool unbiased=True, bool keepdim=False) -> (Tensor, Tensor)
Jul 07 19:50:17 	aten::std_mean.names_dim(Tensor self, str[1] dim, bool unbiased=True, bool keepdim=False) -> (Tensor, Tensor)
Jul 07 19:50:17 	aten::std(Tensor self, bool unbiased=True) -> (Tensor)
Jul 07 19:50:17 	aten::std.dim(Tensor self, int[1] dim, bool unbiased=True, bool keepdim=False) -> (Tensor)
Jul 07 19:50:17 	aten::std.names_dim(Tensor self, str[1] dim, bool unbiased=True, bool keepdim=False) -> (Tensor)
Jul 07 19:50:17 	aten::std.names_out(Tensor self, str[1] dim, bool unbiased=True, bool keepdim=False, *, Tensor(a!) out) -> (Tensor(a!))
Jul 07 19:50:17 	aten::std.out(Tensor self, int[1] dim, bool unbiased=True, bool keepdim=False, *, Tensor(a!) out) -> (Tensor(a!))
Jul 07 19:50:17 	aten::var(Tensor self, bool unbiased=True) -> (Tensor)
Jul 07 19:50:17 	aten::var.dim(Tensor self, int[1] dim, bool unbiased=True, bool keepdim=False) -> (Tensor)
Jul 07 19:50:17 	aten::var.names_dim(Tensor self, str[1] dim, bool unbiased=True, bool keepdim=False) -> (Tensor)
Jul 07 19:50:17 	aten::var.names_out(Tensor self, str[1] dim, bool unbiased=True, bool keepdim=False, *, Tensor(a!) out) -> (Tensor(a!))
Jul 07 19:50:17 	aten::var.out(Tensor self, int[1] dim, bool unbiased=True, bool keepdim=False, *, Tensor(a!) out) -> (Tensor(a!))
Jul 07 19:50:17 	aten::var_mean(Tensor self, bool unbiased=True) -> (Tensor, Tensor)
Jul 07 19:50:17 	aten::var_mean.dim(Tensor self, int[1] dim, bool unbiased=True, bool keepdim=False) -> (Tensor, Tensor)
Jul 07 19:50:17 	aten::var_mean.names_dim(Tensor self, str[1] dim, bool unbiased=True, bool keepdim=False) -> (Tensor, Tensor)
```

Reply:
Ah, right. I've updated the date and rebased on a newer master.